addons/website_event/models/event_event.py

```diff
@@ -65,8 +65,7 @@ def _default_question_ids(self):
         readonly=False, store=True)
     location_menu_ids = fields.One2many(
         "website.event.menu", "event_id", string="Location Menus",
-        domain=[("menu_type", "=", "location_menu")])
-    address_name = fields.Char(related='address_id.name')
```

Comment:
Issue when fixing the conflict, you removed a field ^^

Reply:
ups 🤫